nemo/core/optim/optimizer_with_main_params.py

```diff
@@ -187,13 +188,16 @@ def __init__(
         # used with tensor parallel only (no pipeline parallelism)
         # be careful, weight update cannot start until all async grad AR works are done
         self._async_grad_allreduce = async_grad_allreduce
+        self._grad_divisor = 1 / float(get_data_parallel_world_size())
```

Comment:
We probably don't need `float()` here? python 3 div should be float anyway right?

Reply:
Oh, yes. This seems redundant. Let me fix it. Thanks.

Reply:
fixed. check it out.